demo-node/entrypoint.bash

```diff
@@ -43,20 +43,16 @@ curl -X POST \
 #  -d "{\"name\": \"Live CVE Notifications from the NIST API\", \"definition\": \"${cve_pipe}\", \"start_when_created\": true}" \
 #  http://127.0.0.1:5160/api/v0/pipeline/create
 
-# The shell operator decompresses the data and writes it to `read suricata` on
-# the fly.
 # !! Not started because the data has already been imported while building the image.
-suricata_pipe="shell 'bash -c \\\"curl -s -L https://storage.googleapis.com/tenzir-datasets/M57/suricata.tar.zst | tar -x --zstd --to-stdout\\\"' | read suricata | where #schema != \\\"suricata.stats\\\" | import"
+suricata_pipe="tenzir 'load https https://storage.googleapis.com/tenzir-datasets/M57/suricata.json.zst | decompress zstd | read suricata | import'"
 curl -X POST \
   -H "Content-Type: application/json" \
   -d "{\"name\": \"M57 Suricata Import\", \"definition\": \"${suricata_pipe}\", \"start_when_created\": false}" \
   http://127.0.0.1:5160/api/v0/pipeline/create
 
-# The shell operator "streams" the data into a `Zeek/` directory on disk, calls
-# `sync` to flush the OS write buffers, and finally sends the concatenation of
-# the logs into `read zeek-tsv`.
 # !! Not started because the data has already been imported while building the image.
-zeek_pipe="shell 'bash -c \\\"curl -s -L https://storage.googleapis.com/tenzir-datasets/M57/zeek.tar.zst | tar -x --zstd; sync; cat Zeek/*.log\\\"' | read zeek-tsv | import"
+zeek_pipe="tenzir 'load https https://storage.googleapis.com/tenzir-datasets/M57/zeek-all.log.zst | decompress zstd | read zeek-tsv | import'"
```

Comment:
Since we soon will show the definition on multiple lines, can you `s/ | /\\n| /g` for both this and the Suricata pipeline?

Reply:
you mean replacing with 
```
tenzir 'load https https://storage.googleapis.com/tenzir-datasets/M57/zeek-all.log.zst \n| decompress zstd \n| read zeek-tsv \n| import'
```
?

Reply:
Yes, although I think you can remove the space before the newline as well.

Reply:
doesn't hurt much and it helps readability of the code so I would rather leave them

Reply:
Fine, I don't feel too strongly about them.